staging/src/k8s.io/client-go/transport/round_trippers.go

```diff
@@ -72,7 +72,7 @@ func HTTPWrappersForConfig(config *Config, rt http.RoundTripper) (http.RoundTrip
 func DebugWrappers(rt http.RoundTripper) http.RoundTripper {
 	switch {
 	case bool(klog.V(9).Enabled()):
-		rt = NewDebuggingRoundTripper(rt, DebugCurlCommand, DebugDetailedTiming, DebugResponseHeaders)
+		rt = NewDebuggingRoundTripper(rt, DebugCurlCommand, DebugRequestHeaders, DebugResponseStatus, DebugResponseHeaders, DebugDetailedTiming)
```

Comment:
this line has the same parameters since 1.4 (at least is what git blame tell me)

https://github.com/kubernetes/kubernetes/blame/82ebcd171909c4cd6007007fc07b025d0950153d/staging/src/k8s.io/client-go/transport/round_trippers.go#L71

I couldn't find that level 9 had `DebugRequestHeaders, DebugResponseStatus` 

The problem with logging with too many data is that the logger interleaves the output, doesn't log sequentially and make the output hard to read

Reply:
`DebugRequestHeaders` you are right, and I should remove it. 

But ResponseStatus was actually part of DebugUrlTiming:

https://github.com/kubernetes/kubernetes/blob/release-1.22/staging/src/k8s.io/client-go/transport/round_trippers.go#L454
```
reqInfo.ResponseStatus
```

This got removed/changed and lost with time

Reply:
ok, now I got it, sorry 

When I've added detailed timing https://github.com/kubernetes/kubernetes/commit/72c35be086fed5f70fd11dc96ca34298573bff05
I didn't logged the `reqInfo.RequestVerb, reqInfo.RequestURL, reqInfo.ResponseStatus` 

Maybe 
```
rt = NewDebuggingRoundTripper(rt, DebugCurlCommand, DebugURLTiming, DebugDetailedTiming, DebugResponseHeaders)
```

is more correct and backwards compatible, since only adds the information that I inadvertently removed

Reply:
lgtm, changed here :)